pkg/tfbridge/detailed_diff.go

```diff
@@ -427,16 +428,77 @@ func (differ detailedDiffer) makePropDiff(
 	}
 }
 
+func makeListAttributeDiff(
+	path propertyPath, old, new []resource.PropertyValue,
+) map[detailedDiffKey]*pulumirpc.PropertyDiff {
+	diff := make(map[detailedDiffKey]*pulumirpc.PropertyDiff)
+	lcs := objchange.LongestCommonSubsequence(old, new, func(x, y resource.PropertyValue) bool {
+		return x.DeepEquals(y)
+	})
+
+	//nolint:lll
+	// Adapted from https://github.com/opentofu/opentofu/blob/cb2e9119aa75eeb8e1fa175e2b7a205a4fef129f/internal/command/jsonformat/collections/slice.go#L39
```

Comment:
Why? Do you grok what's going on here? It's using LCS but then it's doing some extra work in here to compute edits, which I assume is not specific to TF but is generic CS thing? Doesn't godifft give you that + testing?